tests/phpunit/test-class-wc-stripe-upe-payment-gateway.php

```diff
@@ -221,6 +221,7 @@ private function get_order_details( $order ) {
 			'order_id'       => $order_id,
```

Comment:
```suggestion
			'order_id'       => $order_number,
```
In order to match [this](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/b55ed7abce8e2a047976245c1cd0db7180b6cbed/includes/abstracts/abstract-wc-stripe-payment-gateway.php#L484) and fix the failed tests.

Reply:
Good catch, Mayisha! Applied ✅